cmd/common.go

```diff
@@ -28,31 +28,27 @@ var (
 )
 
 // workspaceExists checks if workspace exists in Kong.
-func workspaceExists(ctx context.Context, config utils.KongClientConfig) (bool, error) {
-	if config.Workspace == "" {
+func workspaceExists(ctx context.Context, rootConfig utils.KongClientConfig, workspaceName string) (bool, error) {
```

Comment:
To avoid bugs, we can make this function work on any `KongClientConfig`, not only one with workspace unset.

To do that:
- rename the argument `rootConfig` to `config`, and pass it by pointer (`*utils.KongClientConfig`)
- in the function, make the config a root config explicitly:
    ```go
    rootConfig := config.ForWorkspace("")
    ```